### DISCUSSION TITLE
UI library

### DISCUSSION THREAD

**Discussion by brentneave** [jorgebucaran/hyperapp#1041] [Ideas]
Is there a standard UI library of pre-built components for Hyperapp? Would such a thing be useful?

**Comment by jorgebucaran** [comment]
No there isn't, and yes, it would be tremendously useful.

**Comment by zaceno** [comment]
IMO an “official standard” ui library is out of scope for core Hyperapp. But I’d really like to see the community come up with a few alternatives that could inspire each other!
